publish-nuget/action.yml

```diff
@@ -23,10 +23,34 @@ inputs:
   dotnetVersion:
     description: "Version of dotnet to use."
     required: true
+  publishMode:
+    description: "Publish mode to use (always | when_any | only_changed). [always] will publish all packable projects regardless of changes. [when_any] will publish all packable projects when any of them has changes. [only_changed] will publish only packable projects with changes."
+    required: true
+    default: "only_changed"
```

Comment:
I'd like to confirm with the team if we should keep this as default or maybe better use `when_any`.
For `common` projects I'd manually set to `always` so these specific packages would always have the same version (easier to maintain I guess).